packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/zoe-upgrade/zoe-upgrade-driver.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+. ./upgrade-test-scripts/env_setup.sh
+
+set -euo pipefail
+
+here='upgrade-test-scripts/agoric-upgrade-11/zoe-upgrade'
+
+agd --chain-id=agoriclocal \
+  tx gov submit-proposal swingset-core-eval \
+  ${here}/zoe-upgrade-permit.json ${here}/zoe-upgrade-script.js \
+  --title="Zoe Upgrade" --description="zoe upgrade test" \
+  --deposit=10000000ubld \
+  --gas=auto --gas-adjustment=1.2 \
+  --yes -o json --from=validator --keyring-backend=test -b sync
```

Comment:
I think you'll need `-b block` or else your query on the next line will race and fail to display the submitted proposal.

```suggestion
  --yes -o json --from=validator --keyring-backend=test -b block
```